Resources/Prototypes/SimpleStation14/Entities/Clothing/Head/hardsuit-helmets.yml

```
@@ -13,3 +13,27 @@
     highPressureMultiplier: 0.3
     lowPressureMultiplier: 1000
 
+- type: entity
+  parent: ClothingHeadHardsuitWithLightBase
+  id: ClothingHeadHelmetHardsuitHoP
+  noSpawn: true
+  name: head of personnel's plated hardsuit helmet
+  description: Fancy helmet for a fancy hardsuit. Doesn't offer much damage protection.
+  components:
+  - type: Sprite
+    sprite: SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi
+  - type: Clothing
+    sprite: SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi
+  - type: PointLight
+    color: "#ff2034"
+  - type: PressureProtection
+    highPressureMultiplier: 0.525
+    lowPressureMultiplier: 10000
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.95
+        Slash: 0.95
+        Piercing: 0.95
+        Heat: 0.9
+        Radiation: 0.9
```

```suggestion
        Radiation: 0.6
```
Even a _bad_ hardsuit should probably still be really good for radiation :P